README.rst

```diff
@@ -196,7 +196,7 @@ Development
 Patches are welcome.
 
 To run tests, install ``tox`` and run ``tox`` from the project root.
-This will run the tests in Python 3.7, 3.8 and 3.9 against
+This will run the tests in Python 3.9, 3.10 and 3.11 against
```

Comment:
Hmh thinking of it now, it was maybe a little premature to remove 3.8 as it will only become EOL in a month or two from now, I was perhaps too eager to future-proof this right away;D

Let me know if you'd like to keep 3.8 in the matrix…